internal/gatewayapi/clienttrafficpolicy.go

```diff
@@ -725,9 +740,27 @@ func (t *Translator) translateListenerTLSParameters(policy *egv1a1.ClientTraffic
 			}
 		}
 
+		// forwardClientCert := &ir.ForwardClientCert{}
```

Comment:
can we remove these comments? or is there any changes left to address?

Reply:
miss this, thanks